apps/system/js/shortcuts.js

```diff
@@ -0,0 +1,60 @@
+/* global BaseModule, ScreenManager */
+'use strict';
+(function() {
+  /* For hardware key handling that doesn't belong to anywhere */
+  var Shortcuts = function() {};
+  Shortcuts.EVENTS = [
+    'keyup'
+  ];
+  BaseModule.create(Shortcuts, {
+    name: 'Shortcuts',
+
+    _start: function() {
+      window.addEventListener('touchstart', this, true);
+      window.addEventListener('touchend', this, true);
+      window.addEventListener('mousedown', this, true);
+      window.addEventListener('mouseup', this, true);
+    },
+    /* === XXX Bug 900512 === */
```

Comment:
yes we did ! (it's a perf issue for APZC in the main process and we don't have the issue on the flame)